### DISCUSSION TITLE
Mapping resources not work

### DISCUSSION THREAD

**Discussion by YohPaN** [FabricMC/fabric-api#3715] [Mod Dev Support]
### Description

I'm doing a Fabric mod from a fork of a forge mod. I did all the necessary configuration in my build.gradle, then I use fabric loader and API. I depend on create (fabric version) and it work with architectury and Registrate (Fabric version). So fabric come with both so everything is fine, the game can be launch and I come into the menu. But when I start a world, during the initialization of my world, I have lot of error messages that tell me "unknown item" about each custom items, block, blockentity, textures, ect. 
I have debug my codebase and my registrate handle all of my blocks, items, ect.. but it's like fabric don't know where to find JSON for textures, block, item, loottable, ect.
Based on my comprehension of fabric, we don't need to tell him where to find resources.

**Comment by davidromrell** [comment]
first step check this is not an architectury or registrate issue, then add new post if it isn't.
